### PR TITLE
initial commit to projects view

### PR DESCRIPTION
**Pull request by jtallen**
Added Projects View
* Added Projects to navbar
* Added images for Project
* Created custom Project Cards to display projects with image, title, description, and links
* Stubbed dummy data for Projects view

### CODE REVIEW

File: components/Projects.js [hunk 1]
```diff
@@ -9,41 +9,39 @@ export default function Projects() {
...
-            className="transform hover:scale-125 transition duration-2000 ease-out object-cover h-full w-full"
-          />
-        </div>
-        <h1 className="absolute top-10 left-10 text-gray-50 font-bold text-xl bg-red-500 rounded-md px-2">
-          {title}
-        </h1>
-        <h1 className="absolute bottom-10 left-10 text-gray-50 font-bold text-xl">
-          {number.length === 1 ? "0" + number : number}
-        </h1>
+    <div className="rounded flex flex-col justify-between transform hover:scale-105 transition duration-200 shadow-md my-4 mx-4">
```
Comment (jtallen) [inline components/Projects.js]: I like this kind of emphasis behavior (in this case scrolling over the card slightly scales it up) but it looks a little janky with the text scaling up, open to suggestions for other ways to do this!

File: components/Projects.js [hunk 2]
```diff
@@ -9,41 +9,39 @@ export default function Projects() {
...
+      <img className="w-full shadow-sm" src={project.imgUrl}></img>
+      <div className="px-6 py-4 justify-self-start">
```
Comment (jtallen) [inline components/Projects.js]: I'm trying to get the text to stick to the bottom of the image. From the tailwind docs it seems like `justify-self-start` should accomplish this, but it doesn't seem to be working. Hoping you may have some insight here!

File: components/Projects.js [hunk 3]
```diff
@@ -9,41 +9,39 @@ export default function Projects() {
...
+          className="bg-gray-200 hover:bg-gray-300 px-3 py-1 border border-gray-400 rounded-sm text-sm font-semibold text-gray-700 my-2 ml-3 mr-1"
+        >
+          GitHub
+        </a>
+        <a
+          href={project.siteUrl}
+          className="bg-gray-200 hover:bg-gray-300 px-3 py-1 border border-gray-400 rounded-sm text-sm font-semibold text-gray-700 my-2 ml-1 mr-3"
+        >
+          Live Site
+        </a>
```
Comment (jtallen) [inline components/Projects.js]: Not sure about what the best way to do these buttons with external links is, they're simple and functional now but I would like to make them look a bit more pleasing.

**Review by jtallen — COMMENTED** [review]
Would love some quick insight on a few of these issues I ran into, thanks @cchanningallen!

File: components/Projects.js (hunk 2 above)
Comment (jtallen) [reply components/Projects.js]: image

File: components/Projects.js (hunk 3 above)
Comment (jtallen) [reply components/Projects.js]: image

**Comment by jtallen** [comment]
image

Image showing new projects view with slightly scaled down cards/window.

File: components/Projects.js [hunk 4]
```diff
@@ -9,41 +9,39 @@ export default function Projects() {
...
+          <ProjectCard projectID={index} />
+        ))}
       </div>
     </section>
   );
 }
 
-const ProjectCard = ({ title, link, imgUrl, number }) => {
+const ProjectCard = ({ projectID }) => {
+  const project = userData.projects[projectID];
```
Comment (cchanningallen) [inline components/Projects.js]: Why not just pass the `project` as a prop on line 14? Then you don't have to go dig back into the array

File: components/Projects.js (hunk 1 above)
Comment (cchanningallen) [reply components/Projects.js]: Nice! I think you have to be pretty careful with scaling - especially with something this big. It's also worth being careful not to mislead a user into thinking something's interactive when it's not (in this case, the card that scales can't be interacted with, which feels a bit strange)

Instead, I'd suggest removing the scaling behavior from the entire card and instead apply it to just the link buttons - see illustrated in my example PR.

File: components/Projects.js (hunk 2 above)
Comment (cchanningallen) [reply components/Projects.js]: Yep - I think it has to do with the images being different heights.

I'd suggest you overlay the title / description block over the _top_ of the image, though. See my draft PR for implementaiton.

File: components/Projects.js (hunk 3 above)
Comment (cchanningallen) [reply components/Projects.js]: I hear ya! Took a stab at prettying them up - see my draft PR

**Review by cchanningallen — CHANGES_REQUESTED** [review]
I _really_ like this overall - great work so far!

I took a stab at a slightly different aesthetic treatment in this draft PR. Take a look and feel free to use any/all of it here.

Additionally, I bumped your package.json packages to later versions - the OG repo was on outdated versions of Next, Tailwind, and React

**Pull request merged**